package.json

```diff
@@ -28,6 +28,7 @@
 		"@types/jest": "^27.4.1",
 		"@types/lodash": "^4.14.179",
 		"@types/react": "^18.0.15",
+		"@types/react-beautiful-dnd": "^13.1.2",
```

Comment:
I can't add this comment in the yarn.lock file because it's too large, but we shouldn't be committing the yarn.lock file with regular PRs. If you need to install and work with new packages you can without having to install them locally in your computer via docker... for example with this package, you manually add the package and version you need here in the package json and then when you run the `sudo npm run docker` in the putty, it will install that dependency but just in the docker, that way you can work with what you need (it will still probably give you type warnings but just in vscode, which is expected). 

We have a script in place called "sync-state" which basically takes all the yarn.lock from the docker and "copies" it to the local machine but with a new lock version. This script should only be run once we fully merge a PR because if not then we'll get tons of merge conflicts because of all the differences between develop's yarn lock and this (don't worry you don't have to mess with this, this is something I do when merging but just FYI).

TL;DR please re-do this PR without committing the yarn.lock